src/CalculatorUITestFramework/StandardCalculatorPage.cs

```diff
@@ -45,5 +60,177 @@ public string GetCalculatorResultText()
         {
             return this.CalculatorResult.Text.Replace("Display is", string.Empty).Trim();
         }
+        /// <summary>
+        /// Gets the text from the display control in KOT mode and removes the narrator text that is not displayed in the UI.
+        /// </summary>
+        /// <returns>The string shown in the UI.</returns>
+        public string GetKOTCalculatorResultText()
```

Comment:
I think CalculatorResult could be it's own class and you could have this as another method in it.

Reply:
In https://github.com/microsoft/calculator/pull/788/commits/ded929f19a56f1eef397c5864d4ec6fde59e78ab